vscode/microsoft-kiota/src/extension.ts

```diff
@@ -143,13 +143,13 @@ export async function activate(
 
         let languagesInformation = await getLanguageInformation(context);
         let availableStateInfo: Partial<GenerateState>;
-        if(deepLinkParams){
-          if (!deepLinkParams["name"] && openApiTreeProvider.apiTitle ){
+        if (deepLinkParams) {
```

Comment:
```suggestion
        if (Object.Keys(deepLinkParams).length > 0) {
```

Reply:
Since you have initialized the deepLinkParams at the top and is no longer undefined by default.